runtime/mainnet/src/config/monetary.rs

```diff
@@ -0,0 +1,439 @@
+use pop_runtime_common::UNIT;
+use sp_runtime::traits::AccountIdConversion;
+
+use crate::{
+	parameter_types, AccountId, Balance, Balances, ConstU32, ConstU8, ConstantMultiplier, PalletId,
+	ResolveTo, Runtime, RuntimeEvent, RuntimeFreezeReason, RuntimeHoldReason,
+	SlowAdjustingFeeUpdate, System, VariantCountOf, EXISTENTIAL_DEPOSIT,
+};
+
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	// src: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/constants/src/polkadot.rs#L70
+	(items as Balance * 20 * UNIT + (bytes as Balance) * 100 * fee::MILLI_CENTS) / 100
+}
+
+/// Constants related to Polkadot fee payment.
+/// Source: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/constants/src/polkadot.rs#L65C47-L65C58
+pub mod fee {
+	use frame_support::{
+		pallet_prelude::Weight,
+		weights::{
+			constants::ExtrinsicBaseWeight, FeePolynomial, WeightToFeeCoefficient,
+			WeightToFeeCoefficients, WeightToFeePolynomial,
+		},
+	};
+	use pop_runtime_common::{Balance, MILLI_UNIT};
+	use smallvec::smallvec;
+	pub use sp_runtime::Perbill;
+
+	pub const CENTS: Balance = MILLI_UNIT * 10; // 100_000_000
```

Comment:
This makes more sense to me personally:
```suggestion
	pub const CENTS: Balance = UNIT / 100; // 100_000_000
```